_specs/ecip-1066.md

```diff
@@ -11,24 +11,21 @@ created: 2019-10-07
+| Mystique <sub>London</sub>                   | 1452500   | 2022-02-12 | [EIP-3529](https://eips.ethereum.org/EIPS/eip-3529) <br/> [EIP-3541](https://eips.ethereum.org/EIPS/eip-3541)                                                                                                                                                                                                                                                                                                                                                                    | [ECIP-1104](https://ecips.ethereumclassic.org/ECIPs/ecip-1104) |                                                   |  |
+| Magneto <sub>Berlin</sub>                    | 13189133  | 2021-07-23 | [EIP-2565](https://eips.ethereum.org/EIPS/eip-2565) <br/> [EIP-2929](https://eips.ethereum.org/EIPS/eip-2929) <br/> [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) <br/> [EIP-2930](https://eips.ethereum.org/EIPS/eip-2930)                                                                                                                                                                                                                                                | [ECIP-1103](https://ecips.ethereumclassic.org/ECIPs/ecip-1103) |                                                   |  |
+| Thanos                                       | 11700000  | 2020-11-28 | [ECIP-1099](https://ecips.ethereumclassic.org/ECIPs/ecip-1099)                                                                                                                                                                                                                                                                                                                                                                                                                   |                                                                |                                                   |  |
+| MESS                                         | 11380000  | 2020-10-10 | [ECBP-1100](https://ecips.ethereumclassic.org/ECIPs/ecip-1100)                                                                                                                                                                                                                                                                                                                                                                                                                   |                                                                |                                                   |  |
```

Comment:
So yeah - let's just remove this line, please, unless we specifically have ethereum/tests for it?
It is not a part of the consensus protocol.
Also (hopefully) it will be gone soon enough anyway.

Reply:
Done with https://github.com/ethereumclassic/ECIPs/pull/499#pullrequestreview-1167038662